Dockerfile

```diff
@@ -0,0 +1,48 @@
+#  Copyright (C) 2024 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# NAV2 extensions build file
+FROM ros:humble-ros-base AS base_image
+FROM base_image AS setup
+ARG GIT_BRANCH="develop"
+
+RUN mkdir /root/c1t_ws/
+RUN mkdir /root/c1t_ws/src/
+RUN mkdir /root/c1t_ws/src/navigation2_extensions
```

Comment:
I think we can combine these RUN commands into single as RUN mkdir -p /root/c1t_ws/src/navigation2_extensions